### PR TITLE
Remove default solver in solver.jl (use dummy) and revert expdesign back

### PR DESCRIPTION
**Pull request by chriscoey**
default solvers have been removed in JuMP. Pajarito now seems to work with JuMP 0.16

with JuMP now detecting symmetry with epsilon, no need to add extra variables and constraints in exp design models

### CODE REVIEW

File: src/solver.jl [hunk 1]
```diff
@@ -87,6 +94,16 @@ function PajaritoSolver(;
     tol_prim_infeas = 1e-6,
     )
 
+    if mip_solver == UnsetSolver()
+        error("No MIP solver specified (set mip_solver)\n")
```
Comment (mlubin) [inline src/solver.jl]: "set mip_solver" seems a bit ambiguous. "use mip_solver keyword argument"?

Comment (chriscoey) [inline src/solver.jl]: I have a lot of errors in conic algorithm that tell users to try this or that with options. this is a lot easier to understand than those

Comment (chriscoey) [inline src/solver.jl]: we can create a new issue to give more detailed error and warning messages if you like

Comment (mlubin) [inline src/solver.jl]: Ok

**Pull request merged**